### PR TITLE
WIP Fix invitation mail body

### PR DESCRIPTION
**Pull request by victorbalan**

### CODE REVIEW

File: platform-server/src/main/resources/application-development.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 coduno:
-  url: http://localhost:6060
+  url: http://localhost:5000
```
Comment (flowlo) [inline platform-server/src/main/resources/application-development.yml]: Wut?

Comment (victorbalan) [inline platform-server/src/main/resources/application-development.yml]: by default gulp serves on 5000. Doesn't it do the same for you?

Comment (Exordian) [inline platform-server/src/main/resources/application-development.yml]: it is, i like that change, i already had that in my -local.yml :)

File: platform-server-kernel/src/main/java/uno/cod/platform/server/core/service/mail/MailService.java [hunk 2]
```diff
@@ -55,7 +55,7 @@ public void sendMail(final String recipientName,
         message.setTo(recipientEmail);
 
         // Create the HTML body using Thymeleaf
-        final String htmlContent = this.templateEngine.process(template, ctx);
+        final String htmlContent = this.templateEngine.process(template, ctx).replaceAll("&amp;", "&");
```
Comment (victorbalan) [inline platform-server-kernel/src/main/java/uno/cod/platform/server/core/service/mail/MailService.java]: haha :)) i told you on slack.

File: platform-server/src/main/resources/application-development.yml (hunk 1 above)
Comment (flowlo) [inline platform-server/src/main/resources/application-development.yml]: OK

File: platform-server-kernel/src/main/java/uno/cod/platform/server/core/service/mail/MailService.java (hunk 2 above)
Comment (flowlo) [inline platform-server-kernel/src/main/java/uno/cod/platform/server/core/service/mail/MailService.java]: So this needs a proper fix then.

Comment (rogi1609) [inline platform-server-kernel/src/main/java/uno/cod/platform/server/core/service/mail/MailService.java]: ran into this problem too^^

**Comment by lorenzleutgeb** [comment]
Superseded by #194

**Pull request closed without merging**